### PR TITLE
optional nsx header fields

### PR DESCRIPTION
**Pull request by Saksham20**
## Motivation

Some nsx files do not have the keys that `BlackrockRecordingExtractorInterface` looks for in constructing the metadata. This PR makes it optional so that it does not throw an error when not found. 

@luiztauffer you had ported this code in its first commit so can you give a quick glance. 

## Checklist

- [x] Have you checked our [Contributing](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/docs/contribute.rst) document?
- [x] Have you ensured the PR description clearly describes the problem and solutions?
- [x] Have you checked to ensure that there aren't other open or previously closed [Pull Requests](https://github.com/catalystneuro/nwb-conversion-tools/pulls) for the same change?
- [x] If this PR fixes an issue, is the first line of the PR description `fix #XXX` where `XXX` is the issue number?

### CODE REVIEW

File: nwb_conversion_tools/datainterfaces/ecephys/blackrock/blackrockdatainterface.py [hunk 1]
```diff
@@ -51,18 +51,15 @@ def get_metadata_schema(self):
...
-        comment = nsx_file.basic_header["Comment"]
-
-        # Updates basic metadata from files
-        metadata["NWBFile"] = dict(
-            session_start_time=session_start_time_tzaware.strftime("%Y-%m-%dT%H:%M:%S"),
-            session_description=comment,
-        )
+        if "TimeOrigin" in nsx_file.basic_header:
+            session_start_time = nsx_file.basic_header["TimeOrigin"]
+            session_start_time_tzaware = pytz.timezone("EST").localize(session_start_time)
```
Comment (bendichter) [inline nwb_conversion_tools/datainterfaces/ecephys/blackrock/blackrockdatainterface.py:59]: why are we assuming EST?

Comment (luiztauffer) [reply nwb_conversion_tools/datainterfaces/ecephys/blackrock/blackrockdatainterface.py:59]: as I remember it, there is no tz info in blackrock files, we need to assume something

**Comment by luiztauffer** [comment]
@Saksham20 thanks for those fixes!

**Review by luiztauffer — APPROVED** [review]

**Comment by bendichter** [comment]
Would there be a way of inputting the timezone information?

**Comment by luiztauffer** [comment]
> Would there be a way of inputting the timezone information?

via metadata, but user would have to:
1. get metadata, which would return the right timestamp, but the default EST
2. update the session_start_time with the correct TZ
3. pass this metadata to run_conversion

**Comment by bendichter** [comment]
yeah I suppose that's not that bad

**Comment by bendichter** [comment]
Could we return a non-localized time?

**Comment by luiztauffer** [comment]
> Could we return a non-localized time?

yes, but pynwb will raise a warning at conversion time, I believe

**Comment by bendichter** [comment]
Yes I think it's OK to have a warning. That will automatically localize to the user's timezone, which I think it the best thing to guess by default, and will also throw a warning, indicating that the user should add this information themselves if they want to override this default. When we show a demo using the platform, we can show how to apply a timezone localization.

File: nwb_conversion_tools/datainterfaces/ecephys/blackrock/blackrockdatainterface.py [hunk 2]
```diff
@@ -51,18 +51,15 @@ def get_metadata_schema(self):
...
+            metadata["NWBFile"].update(session_start_time=session_start_time_tzaware.strftime("%Y-%m-%dT%H:%M:%S"))
```
Comment (bendichter) [inline nwb_conversion_tools/datainterfaces/ecephys/blackrock/blackrockdatainterface.py:60]:
```suggestion
            metadata["NWBFile"].update(session_start_time=session_start_time.strftime("%Y-%m-%dT%H:%M:%S"))
```

**Comment by luiztauffer** [comment]
@Saksham20 thanks! let me run some local tests and I'll merge it soon

**Pull request merged**